src/Transports.AspNetCore/Compatibility/IAsyncDisposable.cs

```diff
@@ -0,0 +1,10 @@
+#if NETSTANDARD2_0 || NETCOREAPP2_1
+
+namespace GraphQL.Server.Transports.AspNetCore;
+
+internal interface IAsyncDisposable
```

Comment:
Just to satisfy compiler? No implementations?

Reply:
Right. The alternative is to put `#if` around references to IAsyncDisposable. Your call.